Automattic/wp-calypso
# Jetpack Error UX: Show banner on My Home when connection is broken

See https://github.com/Automattic/dotcom-forge/issues/3394

## Proposed Changes

Shows an error notice on My Home when Jetpack is unable to connect to the site:

<img width="1688" alt="CleanShot 2023-08-09 at 05 44 27@2x" src="https://github.com/Automattic/wp-calypso/assets/36432/9415ad89-f380-4407-8272-67cfb45fe576">

Triggering logic was introduced in https://github.com/Automattic/wp-calypso/pull/79965

## Testing Instructions

1. Create a new Business site and take it Atomic.
2. Add some PHP code to `~/htdocs/wp-content/mu-plugins/local.php` that causes a fatal error.
3. Navigate to My Home in Calypso.
4. Verify the notice appears as expected.
5. Fix your fatal error.
6. Reload the My Home view.
7. Verify the notice no longer appears.

Flagging @Automattic/serenity for review in case there are other aspects that need to be tested/considered.
